spec/openapi.yaml

```diff
@@ -847,31 +566,29 @@ components:
         actual: 22.5
         target: 50.0
         offset: 1.0
-    OctoTemperatureState:
+
+    TemperatureState:
```

Comment:
it would be more compatible if use tool0, tool1, bed
Should history be stored in the printer or client? The Octoprint is in the printer.

Reply:
I think MINI cannot provide temperature history. So the history will be kept only in PCL, same as is now. BTW chamber is octo compatible https://docs.octoprint.org/en/master/api/printer.html?highlight=chamber#retrieve-the-current-chamber-state you just need to have chamber enabled in https://www.weedotina2.com/lib/exe/fetch.php?media=octoprint_f152s_03.png

Reply:
It was only subnode of state (state->telemetry). But I changed it as you wanted.